src/main/java/Duke/Tasks.java

```diff
@@ -0,0 +1,29 @@
+package Duke;
+
+public class Tasks {
+    String taskName;
+    boolean done;

```

Comment:
Booleans should be made to sound like booleans, as per the Java coding standard.